portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,185 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<html lang="en">
```

Comment:
These formatting looks a bit odd, is there a way to fix it with indentation?

Reply:
I will make sure to check all of the indentation in my current code. My current version does not contain this piece of code anymore.